__tests__/endpoints.test.js

```diff
@@ -104,6 +104,22 @@ describe("GET: /api/articles", () => {
         });
       });
   });
+  test('GET: 400 return "Invalid sort_by value" when given invalid sort by', () => {
+    return request(app)
+      .get("/api/articles?sort_by=create_t&order=desc")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid sort_by value");
+      });
+  });
+  test('GET: 400 return "Invalid order value" when given invalid order', () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=dsc")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid order value");
+      });
```

Comment:
We have one untested behaviour here:

It is possible to misspell sort_by or order EG  .get("/api/articles?sock_by=created_at")